src/_includes/banner.html

```diff
@@ -6,9 +6,7 @@
     Looking for inspiration?<br>
     Check out the
     <a href="https://www.youtube.com/watch?v=MI7Rl2umeeU">Flutter
-    Vignettes video</a><br>
-    created by the noted design firm, 
-    <a href="https://gskinner.com/">gskinner</a>, for Flutter Interact.<br>
-    See what's possible in Flutter!
+    Vignettes video</a><br> by the noted design firm 
+    <a href="https://gskinner.com/">gskinner</a> for Flutter Interact.<br>
```

Comment:
I was wrong... "created" is important if we're going to keep " for Flutter Interact". (Do we need to mention Flutter Interact?)